### PR TITLE
Add zuul based molecule jobs

### PR DESCRIPTION
**Pull request by son-vyas**

### CODE REVIEW

**Comment by raukadah** [comment]
recheck

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+- job:
+    name: install-yaml-molecule
+    parent: cifmw-molecule-base
+    required-projects:
+      - openstack-k8s-operators/install_yamls
+      - github.com/openstack-k8s-operators/ci-framework
+      - github.com/openstack-k8s-operators/repo-setup
```
Comment (cjeanner) [inline zuul.d/jobs.yaml]: I'm not sure we actually need that one... ? since this is the very project we're testing?

Comment (son-vyas) [reply zuul.d/jobs.yaml]: yes, We dont require it here.

Comment (cjeanner) [reply zuul.d/jobs.yaml]: nice :).
Maybe squash patches so that only one is actually pushed? Cleaner history in the end ;).

**Comment by raukadah** [comment]
@son-vyas Thank you for the patch. It looks good for time being. It is not properly testing the repo-setup code. We need to modify the molecule job to override the role vars. In case of [repo-setup.](https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/repo_setup/defaults/main.yml#L30). We need to change the value of `cifmw_repo_setup_src`. We will do a follow up on this.

**Review by raukadah — APPROVED** [review]
/lgtm

**Review by raukadah — APPROVED** [review]
/lgtm

**Pull request merged**